doc/adding-your-service.md

```diff
@@ -38,22 +34,6 @@ if its proxy status is green in the Conduit dashboard.
 
 ## Protocol Support
 
-Conduit supports most applications without requiring any configuration on your
-part. To accomplish this, Conduit automatically detects the protocol used on
-each connection. In some cases, however, Conduit's protocol detection can't be
-fully automated and requires some configuration from you.
-
-### MySQL and SMTP
```

Comment:
It's probably warranted to leave some explanation of server-speaks-first protocols and skipping ports. While it's typically not required for mysql or smtp, it may impact other protocols, or these protocols may be used on non-default ports.

Reply:
Ok, I took a stab at this in 6add32c7. Edits welcome.